doc/sphinxman/source/tdscf.rst

```diff
@@ -0,0 +1,1024 @@
+.. #
+.. # @BEGIN LICENSE
+.. #
+.. # Psi4: an open-source quantum chemistry software package
+.. #
+.. # Copyright (c) 2007-2019 The Psi4 Developers.
+.. #
+.. # The copyrights for code used from other parties are included in
+.. # the corresponding files.
+.. #
+.. # This file is part of Psi4.
+.. #
+.. # Psi4 is free software; you can redistribute it and/or modify
+.. # it under the terms of the GNU Lesser General Public License as published by
+.. # the Free Software Foundation, version 3.
+.. #
+.. # Psi4 is distributed in the hope that it will be useful,
+.. # but WITHOUT ANY WARRANTY; without even the implied warranty of
+.. # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+.. # GNU Lesser General Public License for more details.
+.. #
+.. # You should have received a copy of the GNU Lesser General Public License along
+.. # with Psi4; if not, write to the Free Software Foundation, Inc.,
+.. # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+.. #
+.. # @END LICENSE
+.. #
+
+.. include:: autodoc_abbr_options_c.rst
+
+.. index:: TDSCF, TDDFT, RPA, random phase approximation, TDA, Tamm--Dancoff approximation
+.. _`sec:tdscf`:
+
+HF: Hartree--Fock Theory
+========================
+
+.. codeauthor:: Andrew M. James, Daniel G. A. Smith, Ruhee Dcuhna, Roberto Di Remigio and Jeff Schriber
+.. sectionauthor:: Roberto Di Remigio
+
+*Module:* :ref:`Keywords <apdx:tdscf>`, :ref:`PSI Variables <apdx:tdscf_psivar>`, :source:`LIBSCF_SOLVER <psi4/src/psi4/libscf_solver>`
+
+.. _`sec:tdscfintro`:
+
+Single-reference response theory:
+ [Dreuw2005-wp]_
+
+Hamiltonian solver:
+ [stratmann:1998]_
+
+Calculation of spectroscopic observables in RPA and TDA:
+ [Pedersen1995-du]_
+ [Lestrange2015-xn]_
+
+Plotting of OPA and ECD:
+ [Rizzo2011-to]_
+
+Introduction
+~~~~~~~~~~~~
+
+An illustrative example of using the TDSCF functionality is as follows::
+
+    molecule {
+    0 1
+    O        0.000000    0.695000   -0.092486
+    O       -0.000000   -0.695000   -0.092486
+    H       -0.388142    0.895249    0.739888
+    H        0.388142   -0.895249    0.739888
+    }
+
+    set {
+    tdscf_states 10
+    }
+
+    energy('td-scf/cc-pvdz')
+
+
+Known limitations
+~~~~~~~~~~~~~~~~~
+
+Psithon usage
+~~~~~~~~~~~~~
+
+PsiAPI usage
+~~~~~~~~~~~~
+
+Plotting one-photon absorption and electronic circular dichroism spectra
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
```

Comment:
all below is placeholder, I think -- this file was easier to review than I thought :-)

Reply:
Yes, placeholder. I started writing when I noticed that the example I chose (H2O2 with symmetry and DF) didn't actually run...